src/content/en/tools/workbox/modules/workbox-routing.md

```diff
@@ -39,47 +39,52 @@ expressions or Route instances.
 
 ## Matching and Handling in Routes
 
-A "route" in workbox is nothing more than two functions: a “matching” function
-to determine if the route should match a request and a “handling” function,
-which should handle the request and respond with a Response.
+A "route" in workbox is nothing more than two functions: a "matching" function
+to determine if the route should match a request and a "handling" function,
+which should handle the request and respond with a response.
 
-Workbox comes with some helpers that’ll perform the matching and handling for
-you, but if you ever find yourself wanting different behaviour, writing a
+Workbox comes with some helpers that'll perform the matching and handling for
+you, but if you ever find yourself wanting different behavior, writing a
 custom match and handler function is the best option.
 
-A match function will be given a `FetchEvent` and a URL object and you can
-match against a request by returning a truthy value. For a simple example,
-you could match against a specific URL like so:
+A
+[match callback function](/web/tools/workbox/reference-docs/latest/module-workbox-routing#~matchCallback)
+is passed a
+[`FetchEvent`](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent),
```

Comment:
It's not necessarily going to be a `FetchEvent`. Maybe update this to say `ExtendableEvent` and in parenthesis (usually a `FetchEvent`).